kademlia/network.py

```diff
@@ -154,6 +154,26 @@ async def get(self, key):
                                   self.ksize, self.alpha)
         return await spider.find()
 
+    async def get_digest(self, dkey):
```

Comment:
I think it would make most sense to remove lines 145 - 155 and replace them with a call to this method (so there isn't any unnecessary duplication).